compile/ivy/src/test/sbt/compiler/CascadingVersionsSpecifications.scala

```diff
@@ -0,0 +1,50 @@
+package sbt
+package compiler
+
+import org.junit.runner.RunWith
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class CascadingVersionsSpecification extends Specification {
```

Comment:
Did you add this to teh "safeUnitTests" list.  I realize we have been forgetting to add new unit-test projects to this.